README.md

```diff
@@ -1,226 +1,187 @@
-// CTM
-if (cutsTheMustard) {
-    var o = document.createElement('script');
-    o.async = o.defer = true;
-    o.src = 'https://build.origami.ft.com/v2/bundles/js?modules=o-tracking';
-    var s = document.getElementsByTagName('script')[0];
-    if (o.hasOwnProperty('onreadystatechange')) {
-        o.onreadystatechange = function() {
-            if (o.readyState === "loaded") {
-                oTrackinginit();
-            }
-        };
-    } else {
-        o.onload = oTrackinginit;
+If o-tracking is running on a page which has an ftsession cookie, it will automatically add that information to the data it is tracking.
+
+* Automatically pickup ftsession from cookies for you.
+
+``` js
+import oTracking from 'o-tracking';
+
+const config = {
+    context: {
+        // This value is used as a way to identify the high-level product, for example: ft.com, FT app, biz-ops etc.
+        product: 'o-tracking-example',
     }
-    s.parentNode.insertBefore(o, s);
-}
+};
+oTracking.init(config);
 ```
 
-The `oTrackinginit` function, used above, would have function calls to setup o-tracking and likely send a page view event. e.g.
-
+To track a page view for the product you call the `oTracking.page` method.
+Page events automatically track the url and the referrer.
 ```js
-function oTrackinginit() {
-    var oTracking = Origami['o-tracking'];
-    // Setup
-    oTracking.init({...config...});
-    // Page
-    oTracking.page({...config...});
-    // Clicks.
-    oTracking.click.init(category);
-    // Components/Elements views.
-    oTracking.view.init({...opts...});
-}
+const pageConfig = {
+  content: {
+      /*
+        Asset type is meant to describe the main purpose of the page
+        The value can be one of these:
+        - `story` - A story or article
+        - `blog` - A blog post
+        - `front` - A home page or front page
+        - `ad` - An advert.
+        - `image` - An image
+        - `interactive` - An interactive graphic
+        - `report` - A special report
+        - `search` - A search results page
+        - `section` - A section or listing page
+        - `topic` -  A topic landing page
+        - `video` - A video page
+        - `login` - Any login/sign-in page
+        - `stream` - A stream page
+        - `funnel` - Any funnel page
+        - `epaper` - all epaper pages
+        - `rankings` - A rankings page for schools and courses (i.e. on rankings.ft.com). Not the section's hub page
+        - `markets` - Any market, bond, commoditity, stock, currency 'tearsheet' (usually has "/tearsheet/" in URL)
+        - `myft` - All MyFT pages
+        - `account` - All account pages
+        - `membership` - All membership pages
+        - `page` - anything else, not above.
+    */
+    asset_type: "story"
+  }
+};
+oTracking.page(pageConfig);
 ```
 
-### Alternative implementation using require
+To track click events you call the `oTracking.click.init` method and pass it the category you would like the events to have:
```

Comment:
>To track click events you call the `oTracking.click.init` method and pass it the category you would like the events to have:

Could be written more directly:

>Call the `oTracking.click.init` method to track click events. Pass the category you would like the click events to have as an argument:

At first I thought this is an example of [passive voice which we try to avoid](https://origami.ft.com/docs/principles/tone-and-language/#use-the-active-voice), but now I'm not sure.